templates/datalab/datalab-auth-deployment.template.yml

```diff
@@ -144,9 +144,11 @@ kind: Service
 metadata:
   name: datalab-auth-service
   namespace: {{ namespace }}
+  labels:
+    monitor: prometheus
 spec:
   ports:
-    - name: auth
+    - name: http
```

Comment:
Out of interest - do these have to be named to be scraped correctly or is this just standardizing?

Reply:
I think it had to be changed to allow the single service selector to be used as it has to match the second last line of that.